packages/components/src/DataTable/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import { Table } from "@tanstack/react-table";
+import React from "react";
+import styles from "./Footer.css";
+import { Option, Select } from "../Select";
+import { Button } from "../Button";
+import { Text } from "../Text";
+
+interface FooterProps<T> {
+  table: Table<T>;
+}
+
+export function Footer<T extends object>({ table }: FooterProps<T>) {
+  const { pageIndex, pageSize } = table.getState().pagination;
+  const totalRows = table.getCoreRowModel().rows.length;
+  const firstPosition = pageIndex * pageSize + 1;
+  const secondPosition = Math.min(totalRows, pageSize * (pageIndex + 1));
+
+  return (
+    <div className={styles.footerContainer}>
+      <Text>
+        {`Showing ${firstPosition}-${secondPosition} of ${totalRows} items`}
+      </Text>
+
+      <div className={styles.pageNavigation}>
+        <div className={styles.pageNavigationSelect}>
+          <Select
+            value={table.getState().pagination.pageSize}
+            onChange={value => {
+              table.setPageSize(Number(value));
+            }}
+          >
+            {["10", "20", "30", "40", "50"].map(numOfPages => (
```

Comment:
Is it makes sense to add this option as a prop? To make it dynamic? Or assign it to a variable to understand better what is being done here?

Reply:
You're totally correct @danlaurent, thanks for the feedback. Sorry for the late response :/